<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMenuConfig</class>
 <widget class="QDialog" name="QMenuConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>243</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Log directory:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineLogDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbBrowseLog">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Data directory:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineDataDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbBrowseData">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelHist">
       <property name="text">
        <string>Histograms directory:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineHistDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbBrowseHist">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QSpinBox" name="sbRootFileSize">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="singleStep">
        <number>10000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="chbSaveRootFile">
       <property name="text">
        <string>Record events to .root file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Record frequency (min)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Events per .root file:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chbAdvanced">
       <property name="text">
        <string>Advanced mode</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chbSaveHist">
       <property name="text">
        <string>Record histograms</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="sbHistRecord">
       <property name="maximum">
        <number>1440</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbBrowseLog</sender>
   <signal>clicked()</signal>
   <receiver>QMenuConfig</receiver>
   <slot>OnBrowseLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbApply</sender>
   <signal>clicked()</signal>
   <receiver>QMenuConfig</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>QMenuConfig</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbBrowseData</sender>
   <signal>clicked()</signal>
   <receiver>QMenuConfig</receiver>
   <slot>OnBrowseData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbBrowseHist</sender>
   <signal>clicked()</signal>
   <receiver>QMenuConfig</receiver>
   <slot>OnBrowseHist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnBrowseLog()</slot>
  <slot>OnBrowseData()</slot>
  <slot>OnApply()</slot>
  <slot>OnCancel()</slot>
  <slot>OnSave()</slot>
  <slot>OnBrowseHist()</slot>
 </slots>
</ui>
